specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2024-06-30-preview/examples/DedicatedHsm_Update.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parameters": {
+    "resourceGroupName": "hsm-group",
+    "name": "hsm1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2024-06-30-preview",
+    "parameters": {
+      "tags": {
+        "Dept": "hsm",
+        "Environment": "dogfood",
+        "Slice": "A"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "networkProfile": {
+            "subnet": {
+              "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hsm-group/providers/Microsoft.Network/virtualNetworks/stamp01/subnets/stamp01"
+            },
+            "networkInterfaces": [
+              {
+                "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hsm-group/providers/Microsoft.Network/networkInterfaces/hsm_vnic",
+                "privateIpAddress": "1.0.0.1"
+              }
+            ]
+          },
+          "stampId": "stamp01",
+          "provisioningState": "Succeeded",
+          "statusMessage": "DedicatedHsm device is functional."
+        },
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hsm-group/providers/Microsoft.HardwareSecurityModules/dedicatedHSMs/hsm1",
+        "name": "hsm1",
+        "type": "Microsoft.HardwareSecurityModules/dedicatedHSMs",
+        "location": "westus",
+        "sku": {
+          "name": "SafeNet Luna Network HSM A790"
+        },
+        "tags": {
+          "Dept": "hsm",
+          "Environment": "dogfood",
+          "Slice": "A"
+        }
+      }
+    },
+    "202": {
+      "headers": {
+        "location": "https://some.endpoint.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rgdedicatedhsm/providers/Microsoft.HardwareSecurityModules/dedicatedHSMs/dhsm1/"
```

Comment:
>/some.endpoint.com [](http://example.com/codeflow?start=28&length=18)

nit: suggest using consistent endpoints in your examples. Management.azure.com for example

Reply:
updated endpoints in all examples